src/app/builders/components/Table/Cell/BuilderNameCell/BuilderNameCell.tsx

```diff
@@ -1,95 +1,98 @@
+import { isActive } from '@/app/collective-rewards/active-builders'
```

Comment:
at a certain point we are going to delete this file, could we please move this function to a different place.